BackofficeBundle/Resources/public/coffee/modelBackbone.coffee

```diff
@@ -1,27 +1,28 @@
-Site = Backbone.Model.extend({})
-Block = Backbone.Model.extend({})
-BlockCollection = Backbone.Collection.extend(model: Block)
+SiteModel = Backbone.Model.extend({})
+
+BlockModel = Backbone.Model.extend({})
+BlockCollection = Backbone.Collection.extend(model: BlockModel)
```

Comment:
I think that we could use the naming the same way we do for the facade. We only use singular : `NodeCollection` `SiteCollection` ...